src/components/SecondaryPanes/CommandBar.js

```diff
@@ -235,7 +236,10 @@ class CommandBar extends Component {
 
   render() {
     return (
-      <div className="command-bar">
+      <div
+        className="command-bar"
+        style={{ width: this.props.horizontal ? "100%" : "100vw" }}
```

Comment:
I like where this is headed. Is there a commandBar CSS file so we can do it w classes?